### ISSUE TITLE
Using Column Renderer clears the cell value though returning undefined

### ISSUE THREAD

**Issue by marciogurka** [bryntum/support#5857]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=52&t=23402&p=115840#p115840)

Hi All,

Though returning the undefined from the renderer function, we are unable to see the cell values.

On explicitly setting the alwaysClearCell to false has no effect on the rendering mechanism.

The logic works fine with grid but doesn't work with Gantt.

URL: https://bryntum.com/products/gantt/examples/basic/

Gantt Issue:

```
import { Gantt, StringHelper } from '../../build/gantt.module.js?464505';
import shared from '../_shared/shared.module.js?464505';

new Gantt({
    appendTo          : 'container',
    dependencyIdField : 'sequenceNumber',

project : {
    autoLoad  : true,
    transport : {
        load : {
            url : '../_datasets/launch-saas.json'
        }
    }
},

columns : [
    { 
type : 'name', width : 250, alwaysClearCell: false, renderer: ({ value, cellElement }) => {
                cellElement.innerHTML = `<b>${value}</b>`
                cellElement.firstElementChild.style.backgroundColor = 'blue';
            }
 }
    ],

// Custom task content, display task name on child tasks
taskRenderer({ taskRecord }) {
    if (taskRecord.isLeaf && !taskRecord.isMilestone) {
        return StringHelper.encodeHtml(taskRecord.name);
    }
}
});
```

Snapshot:

![error2.png](https://forum.bryntum.com/download/file.php?id=20114&sid=7af0f2aeab0deba9d2f93de5b13af67e)

Working Grid Example: https://bryntum.com/products/grid/examples/renderers/

Snapshot:

![working2.png](https://forum.bryntum.com/download/file.php?id=20115&sid=7af0f2aeab0deba9d2f93de5b13af67e)
![error2.png](https://forum.bryntum.com/download/file.php?id=20114&sid=7af0f2aeab0deba9d2f93de5b13af67e)
![working2.png](https://forum.bryntum.com/download/file.php?id=20115&sid=7af0f2aeab0deba9d2f93de5b13af67e)

**Comment by taauntik** [comment]
asked for the fix here https://forum.bryntum.com/viewtopic.php?p=116340#p116340

**Comment by JockeLindberg** [comment]
Firstly, the alwaysClearCell config isn't doing anything in this case as it's defaults to false (until version 6.0 where it will default to true).

Secondly, the Gantt [Name column](https://bryntum.com/products/gantt/docs/api/Gantt/column/NameColumn) is a TreeColumn which uses a different rendering mechanism than a regular column. The TreeColumn will use the returned data from the provided renderer and use that as a part of it's content. There is definitly a gap in the documentation here, which we will be fixed in bryntum/issues#5967.

Closing this issue as it isn't directly a bug, more something that's not supported at the moment.
